pkg/actionsmetrics/event_reader.go

```diff
@@ -64,6 +64,10 @@ func (reader *EventReader) ProcessWorkflowJobEvent(ctx context.Context, event in
 		keysAndValues = []interface{}{"job_id", fmt.Sprint(*e.WorkflowJob.ID)}
 	)
 
+	if len(e.WorkflowJob.Labels) == 0 {
```

Comment:
have you checked, what is the cases when we have this empty? just to understand what we are dropping?

Reply:
@kyrylomiro I looked into it a bit, and so far, no run labels appear when no runner was started for the job. This happens when:

- the job calls a reusable workflow. the root job doesn't have runners, but sub jobs in the workflow appears and have runners
- the job has an if statement that wasn't fulfilled
- the job wasn't triggered, due to previous dependency

So I think it should be okay to skip over them. I will do a bit more deep dive after lunch

Reply:
Looks like the list above is accurate. So it should be okay to skip those entries for the metrics.